src/model/attributes/relations/MorphMany.ts

```diff
@@ -0,0 +1,102 @@
+import { Schema as NormalizrSchema } from 'normalizr'
+import { Schema } from '../../../schema/Schema'
+import { Element, Collection } from '../../../data/Data'
+import { Query } from '../../../query/Query'
+import { Model } from '../../Model'
+import { Relation, Dictionary } from './Relation'
+
+export class MorphMany extends Relation {
```

Comment:
I am noticing a fair amount of code duplication, at some point we might want to DRY these up once the library is feature complete (1.x).

Reply:
💯